### PR TITLE
B/new tri wnear farm

### PR DESCRIPTION
**Pull request by memacius32**
Add TRI<>WNEAR MCV2 farm

Screen Shot 2022-09-29 at 17 28 28

Screen Shot 2022-09-29 at 17 27 59

### CODE REVIEW

**Review by specialist0x — APPROVED** [review]

**Comment by specialist0x** [comment]
What changes need to be made to the existing `TRI-wNEAR` farm?

image

**Comment by memacius32** [comment]
- Remove isPeriodFInished hardcoded as false in some hooks.
- Add isPeriodFinished as ** true ** to existing mcv1 tri-wnear farm (disables deposit button)

 Screen Shot 2022-09-29 at 18 29 20

File: src/state/stake/useFarmsAPI.ts [hunk 1]
```diff
@@ -49,7 +50,7 @@ export function useFarmsAPI(): StakingTriFarms[] {
       lpAddress,
       rewarderAddress,
       tokens,
-      isPeriodFinished: false,
```
Comment (specialist0x) [inline src/state/stake/useFarmsAPI.ts:52]: I don't think this is needed

File: src/state/stake/user-farms.ts [hunk 2]
```diff
@@ -66,7 +66,7 @@ export function useSingleFarm(version: number): StakingTri {
     return {
       ...activeFarms[version],
       tokens: tokens!,
-      isPeriodFinished: false,
```
Comment (specialist0x) [inline src/state/stake/user-farms.ts:69]: This change isn't needed

File: src/state/stake/user-stable-farms.ts [hunk 3]
```diff
@@ -58,7 +58,7 @@ export function useSingleStableFarm(version: number, stableSwapPoolName: StableS
 
     return {
       ...activeFarms[version],
-      isPeriodFinished: false,
```
Comment (specialist0x) [inline src/state/stake/user-stable-farms.ts:61]: This change isn't needed

**Comment by memacius32** [comment]
- Removed unnecesary destructure of useFarmsApi, now uses spread operator as the other two hooks
- Removed isPeriodFinished unnecesary explicit return in userSingleFarm and useSingleStableFarm

File: src/state/stake/useFarmsAPI.ts [hunk 4]
```diff
@@ -21,52 +21,23 @@ export function useFarmsAPI(): StakingTriFarms[] {
...
 
     return {
-      ID,
-      poolId,
-      stakingRewardAddress,
-      lpAddress,
-      rewarderAddress,
-      tokens,
-      isPeriodFinished,
+      ...activeFarms[index],
```
Comment (specialist0x) [inline src/state/stake/useFarmsAPI.ts:31]: 👏🏽👏🏽👏🏽

**Review by specialist0x — APPROVED** [review]

**Pull request merged**